### PR TITLE
Handle comments on cases.

### PR DESCRIPTION
**Pull request by mbrandonw**

### CODE REVIEW

File: Tests/CasePathsMacrosTests/CasePathableMacroTests.swift [hunk 1]
```diff
@@ -968,4 +968,102 @@ final class CasePathableMacroTests: XCTestCase {
...
+        public struct AllCasePaths: Sequence {
+          public subscript(root: Foo) -> PartialCaseKeyPath<Foo> {
+            switch root {
+            case .bar:
+              return \.bar
+            case .baz:
+              return \.baz
+            case .fizz:
+              return \.fizz
+            case .fizzier/*Comment in case*/:
```
Comment (mbrandonw) [inline Tests/CasePathsMacrosTests/CasePathableMacroTests.swift:1001]: we could even trim this comment, but it's also not hurting anything and just an edge case

File: Tests/CasePathsTests/MacroTests.swift [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import CasePaths
+
+@CasePathable
+private enum Comments {
+  // Comment above case
+  case bar
+  /*Comment before case*/ case baz(Int)
+  case fizz(buzz: String)  // Comment on case
+  case fizzier/*Comment in case*/(Int, buzzier: String)
+}
```
Comment (mbrandonw) [inline Tests/CasePathsTests/MacroTests.swift:10]: added a "static" macro test. we could probably have more of these.

**Review by stephencelis — APPROVED** [review]
